lib/spack/spack/cmd/create.py

```diff
@@ -184,14 +187,27 @@ class CMakePackageTemplate(PackageTemplate):
     """Provides appropriate overrides for CMake-based packages"""
 
     base_class_name = 'CMakePackage'
-
     body = """\
```

Comment:
Also, I'd really like on failure of my process to leave this intact, that's basic Python which is escaping me